phippery/modeling.py

```diff
@@ -72,3 +74,62 @@ def gamma_poisson_model(
         ds_copy = copy.deepcopy(ds)
         ds_copy[new_table_name] = xr.DataArray(counts)
         return (alpha, beta), ds_copy
+
+
+def neg_binom_model(
+    ds,
+    beads_ds,
+    nb_p=2,
```

Comment:
What is nb_p? Maybe we should start specifying parameters and their purpose in the docstring

Reply:
Added this comment:
'nb_p' determines the relationship between mean and variance. Valid values
are 1 and 2 (sometimes called Type-1 and Type-2 Negative Binominal, respectively)